spec/features/two_factor_authentication/change_factor_spec.rb

```diff
@@ -3,7 +3,7 @@
 feature 'Changing authentication factor' do
   describe 'requires re-authenticating' do
     before do
-      sign_up_and_2fa
+      @user = sign_up_and_2fa
```

Comment:
Also, even if we decide it's OK to use `let`, I don't think we have to use it in every test. It might make sense to use in some tests, but not others. If what `let` is defining is the same in every test in a particular file, then it's probably OK to use. If there are different types of `users` within a file, I prefer to either define the user within the `it` block, or use an instance variable in a `before` block.